Userland/Libraries/LibUnicode/CMakeLists.txt

```diff
@@ -14,4 +14,5 @@ set(GENERATED_SOURCES ${CURRENT_LIB_GENERATED})
 
 serenity_lib(LibUnicode unicode)
 target_link_libraries(LibUnicode PRIVATE LibCore)
+target_link_libraries(LibUnicode PRIVATE AllowInterposition)
```

Comment:
You probably also want LibLocale - and I wonder how our weak definitions of the LibC regex functions deals with this (doesn't seem to fail tests?)

Reply:
I'll try to help MaskRay's `-Bsymbolic-non-weak-functions` Binutils patch accepted + backported by us, which should do the right thing ™️